lib/sisimai/bite/email/verizon.rb

```diff
@@ -0,0 +1,265 @@
+              end
+            end
+
+            if readcursor & Indicators[:'message-rfc822'] > 0
+              # After "message/rfc822"
+              if e.empty?
+                blanklines += 1
+                break if blanklines > 1
+                next
+              end
+              rfc822list << e
+
+            else
+              # Before "message/rfc822"
+              next if readcursor & Indicators[:deliverystatus] == 0
+              next if e.empty?
+
+              # Message details:
+              #   Subject: Test message
+              #   Sent date: Wed Jun 12 02:21:53 GMT 2013
+              #   MAIL FROM: *******@hg.example.com
+              #   RCPT TO: *****@vtext.com
+              v = dscontents[-1]
+              if cv = e.match(/\A[ \t]+RCPT TO: (.*)\z/)
+                if v['recipient']
+                  # There are multiple recipient addresses in the message body.
+                  dscontents << Sisimai::Bite.DELIVERYSTATUS
+                  v = dscontents[-1]
+                end
+
+                v['recipient'] = cv[1]
+                recipients += 1
+                next
+
+              elsif cv = e.match(/\A[ \t]+MAIL FROM:[ \t](.+)\z/)
+                #   MAIL FROM: *******@hg.example.com
+                senderaddr = cv[1] if senderaddr.empty?
+
+              elsif cv = e.match(/\A[ \t]+Subject:[ \t](.+)\z/)
+                #   Subject:
+                subjecttxt = cv[1] if subjecttxt.empty?
+
+              else
+                # 550 - Requested action not taken: no such user here
+                v['diagnosis'] = e if e =~ /\A(\d{3})[ \t][-][ \t](.*)\z/
+              end
+            end
+          end
+
+        else
+          # vzwpix.com
+          re1 = {
+            :begin  => %r/\AMessage could not be delivered to mobile/,
+            :rfc822 => %r/\A__BOUNDARY_STRING_HERE__\z/,
+            :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+          }
+          reFailure = {
+            userunknown: %r/No[ ]valid[ ]recipients[ ]for[ ]this[ ]MM/x,
+          }
+          boundary00 = Sisimai::MIME.boundary(mhead['content-type'])
+          if boundary00.size > 0
+            # Convert to regular expression
+            re1['rfc822'] = Regexp.new('\A' + Regexp.escape('--' + boundary00 + '--') + '\z')
+          end
+
+          hasdivided.each do |e|
+            if readcursor.zero?
+              # Beginning of the bounce message or delivery status part
+              if e =~ re1[:begin]
+                readcursor |= Indicators[:deliverystatus]
+                next
+              end
+            end
+
+            if readcursor & Indicators[:'message-rfc822'] == 0
+              # Beginning of the original message part
+              if e =~ re1[:rfc822]
+                readcursor |= Indicators[:'message-rfc822']
+                next
+              end
+            end
+
+            if readcursor & Indicators[:'message-rfc822'] > 0
+              # After "message/rfc822"
```

Comment:
__[rubocop]__
* Style/NumericPredicate: Use `(readcursor & Indicators[:'message-rfc822']).zero?` instead of `readcursor & Indicators[:'message-rfc822'] == 0`. [:ref](https://github.com/bbatsov/ruby-style-guide#predicate-methods)

<!-- Comment by [SideCI](https://sideci.com) -->